### PR TITLE
AutoCorrect files/zh-cn/web/javascript/reference/global_objects/s*

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

**Review by huacnlee — COMMENTED** [review]
已检查修正

File: files/zh-cn/web/javascript/reference/global_objects/set/has/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 <h2 id="Summary">概述</h2>
 
-<p><strong>has() </strong>方法返回一个布尔值来指示对应的值value是否存在Set对象中。</p>
+<p><strong>has() </strong>方法返回一个布尔值来指示对应的值 value 是否存在 Set 对象中。</p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/has/index.html:14]:
```suggestion
<p><strong>has()</strong> 方法返回一个布尔值来指示对应的值 value 是否存在 Set 对象中。</p>
```

File: files/zh-cn/web/javascript/reference/global_objects/set/index.html [hunk 2]
```diff
@@ -15,26 +15,26 @@
 
 <h2 id="简述">简述</h2>
 
-<p><code>Set</code>对象是值的集合，你可以按照插入的顺序迭代它的元素。 Set中的元素只会<strong>出现一次</strong>，即 Set 中的元素是唯一的。</p>
+<p><code>Set</code>对象是值的集合，你可以按照插入的顺序迭代它的元素。Set 中的元素只会<strong>出现一次</strong>，即 Set 中的元素是唯一的。</p>
 
 <h3 id="值的相等">值的相等</h3>
 
-<p>因为 Set 中的值总是唯一的，所以需要判断两个值是否相等。在ECMAScript规范的早期版本中，这不是基于和===操作符中使用的算法相同的算法。具体来说，对于 Set s， +0 （+0 严格相等于-0）和-0是不同的值。然而，在 ECMAScript 2015规范中这点已被更改。有关详细信息，请参阅<a href="https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set#浏览器兼容性">浏览器兼容性</a> 表中的“<em>Key equality for -0 and 0</em>”。</p>
+<p>因为 Set 中的值总是唯一的，所以需要判断两个值是否相等。在 ECMAScript 规范的早期版本中，这不是基于和===操作符中使用的算法相同的算法。具体来说，对于 Set s，+0（+0 严格相等于  -0）和 -0 是不同的值。然而，在 ECMAScript 2015 规范中这点已被更改。有关详细信息，请参阅<a href="https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set#浏览器兼容性">浏览器兼容性</a> 表中的 “<em>Key equality for -0 and 0</em>”。</p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/index.html:22]:
```suggestion
<p>因为 Set 中的值总是唯一的，所以需要判断两个值是否相等。在 ECMAScript 规范的早期版本中，这不是基于和===操作符中使用的算法相同的算法。具体来说，对于 Set，+0（+0 严格相等于  -0）和 -0 是不同的值。然而，在 ECMAScript 2015 规范中这点已被更改。有关详细信息，请参阅<a href="#浏览器兼容性">浏览器兼容性</a>表中的 “<em>Key equality for -0 and 0</em>”。</p>
```

File: files/zh-cn/web/javascript/reference/global_objects/set/index.html [hunk 3]
```diff
@@ -91,18 +91,18 @@ <h3 id="使用Set对象">使用<code>Set</code>对象</h3>
...
 
-mySet.size; // 4, 刚刚移除一个值
+mySet.size; // 4，刚刚移除一个值
 
 console.log(mySet);
 // logs Set(4) [ 1, "some text", {…}, {…} ] in Firefox
 // logs Set(4) { 1, "some text", {…}, {…} } in Chrome</pre>
 
-<h3 id="迭代Set">迭代Set</h3>
+<h3 id="迭代Set">迭代 Set</h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/index.html:103]:
```suggestion
<h3 id="迭代_set">迭代 Set</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/set/index.html [hunk 4]
```diff
@@ -48,28 +48,28 @@ <h2 id="实例方法">实例方法</h2>
...
+ <dd>返回一个新的迭代器对象，该对象包含 <code>Set</code> 对象中的按插入顺序排列的所有元素的值。</dd>
  <dt>{{jsxref("Set.prototype.@@iterator()", "Set.prototype[@@iterator]()")}}</dt>
- <dd>返回一个新的迭代器对象，该对象包含<code>Set</code>对象中的按插入顺序排列的所有元素的值。</dd>
+ <dd>返回一个新的迭代器对象，该对象包含 <code>Set</code> 对象中的按插入顺序排列的所有元素的值。</dd>
 </dl>
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用Set对象">使用<code>Set</code>对象</h3>
+<h3 id="使用_Set_对象">使用 <code>Set</code> 对象</h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/set/index.html:72]:
```suggestion
<h3 id="使用_set_对象">使用 <code>Set</code> 对象</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/@@iterator/index.html [hunk 5]
```diff
@@ -31,7 +31,7 @@ <h3 id="使用iterator"><code>使用[@@iterator]()</code></h3>
 console.log(strIter.next().value); // "\uD835\uDC68"
 </pre>
 
-<h3 id="通过_for..of_使用iterator"><code>通过</code> <code>for..of 使用[@@iterator]()</code></h3>
+<h3 id="通过_for..of_使用iterator"><code>通过</code> <code>for..of 使用 [@@iterator]()</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/@@iterator/index.html:34]:
```suggestion
<h3 id="通过_for..of_使用_iterator">通过 <code>for..of</code> 使用 <code>[@@iterator]()</code></h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/big/index.html [hunk 6]
```diff
@@ -21,11 +21,11 @@ <h3 id="返回值">返回值</h3>
 
 <h2 id="描述">描述</h2>
 
-<p><code>big()</code> 方法会将一个字符串嵌入到<code>&lt;big&gt;标签中:</code> <code>"&lt;big&gt;str&lt;/big&gt;"。</code></p>
+<p><code>big()</code> 方法会将一个字符串嵌入到<code>&lt;big&gt;标签中：</code> <code>"&lt;big&gt;str&lt;/big&gt;"。</code></p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/big/index.html:24]:
```suggestion
<p><code>big()</code> 方法会将一个字符串嵌入到 <code>&lt;big&gt;</code> 标签中：<code>"&lt;big&gt;str&lt;/big&gt;"</code>。</p>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/big/index.html [hunk 7]
```diff
@@ -21,11 +21,11 @@ <h3 id="返回值">返回值</h3>
...
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用big()函数"><code>使用big()函数</code></h3>
+<h3 id="使用big()函数"><code>使用 big() 函数</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/big/index.html:28]:
```suggestion
<h3 id="使用_big_函数">使用 <code>big()</code> 函数</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/big/index.html [hunk 8]
```diff
@@ -53,12 +53,12 @@ <h2 id="规范">规范</h2>
   <tr>
    <td>{{SpecName('ES6', '#sec-string.prototype.big', 'String.prototype.big')}}</td>
    <td>{{Spec2('ES6')}}</td>
-   <td>初始定义。 在JavaScript 1.0中实现。在（规范性）附件B中定义了用于Web浏览器的附加ECMAScript 特性。</td>
+   <td>初始定义。 在 JavaScript 1.0 中实现。在（规范性）附件 B 中定义了用于 Web 浏览器的附加 ECMAScript 特性。</td>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/big/index.html:56]:
```suggestion
   <td>初始定义。在 JavaScript 1.0 中实现。在（规范性）附件 B 中定义了用于 Web 浏览器的附加 ECMAScript 特性。</td>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/blink/index.html [hunk 9]
```diff
@@ -25,7 +25,7 @@ <h2 id="描述">描述</h2>
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用blink()函数"><code>使用blink()函数</code></h3>
+<h3 id="使用blink()函数"><code>使用 blink() 函数</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/blink/index.html:28]:
```suggestion
<h3 id="使用_blink_函数">使用 <code>blink()</code> 函数</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/bold/index.html [hunk 10]
```diff
@@ -17,15 +17,15 @@ <h2 id="语法">语法</h2>
...
+<p>包含 HTML 元素 {{HTMLElement("b")}} 的字符串。</p>
 
 <h2 id="描述">描述</h2>
 
 <p><code>bold()</code> 方法将一个字符串嵌入到&lt;b&gt;&lt;/b&gt;标记中。</p>
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用bold()"><code>使用bold()</code></h3>
+<h3 id="使用bold()"><code>使用 bold()</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/bold/index.html:28]:
```suggestion
<h3 id="使用_bold">使用 <code>bold()</code></h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/charat/index.html [hunk 11]
```diff
@@ -156,9 +156,9 @@ <h3 id="例子：获取所有字符">例子：获取所有字符</h3>
   return [str.charAt(i+1), i+1];
 }</pre>
 
-<h3 id="示例：修复charAt以支持非基本多文种平面（BMP）字符">示例：修复charAt以支持非基本多文种平面（BMP）字符</h3>
+<h3 id="示例：修复charAt以支持非基本多文种平面（BMP）字符">示例：修复 charAt 以支持非基本多文种平面（BMP）字符</h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/charat/index.html:159]:
```suggestion
<h3 id="示例：修复_charat_以支持非基本多文种平面（bmp）字符">示例：修复 charAt 以支持非基本多文种平面（BMP）字符</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/charcodeat/index.html [hunk 12]
```diff
@@ -63,9 +63,9 @@ <h3 id="Example_Using_charCodeAt">使用 <code>charCodeAt()</code></h3>
 
 "ABC".charCodeAt(3) // returns NaN</pre>
 
-<h3 id="使用_charCodeAt_修复字符串中出现的未知的非基本多语言范围（非BMP，non-Basic-Multilingual-Plane）字符">使用 <code>charCodeAt()</code> 修复字符串中出现的未知的非基本多语言范围（非BMP，non-Basic-Multilingual-Plane）字符</h3>
+<h3 id="使用_charCodeAt_修复字符串中出现的未知的非基本多语言范围（非BMP，non-Basic-Multilingual-Plane）字符">使用 <code>charCodeAt()</code> 修复字符串中出现的未知的非基本多语言范围（非 BMP，non-Basic-Multilingual-Plane）字符</h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/charcodeat/index.html:66]:
```suggestion
<h3 id="使用_charCodeAt_修复字符串中出现的未知的非基本多语言范围（非_bmp，non-basic-multilingual-plane）字符">使用 <code>charCodeAt()</code> 修复字符串中出现的未知的非基本多语言范围（非 BMP，non-Basic-Multilingual-Plane）字符</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/charcodeat/index.html [hunk 13]
```diff
@@ -96,7 +96,7 @@ <h3 id="使用_charCodeAt_修复字符串中出现的未知的非基本多语言
 }
 </pre>
 
-<h3 id="使用_charCodeAt_修复字符串中出现的已知的非BMP字符">使用 <code>charCodeAt()</code> 修复字符串中出现的已知的非BMP字符</h3>
+<h3 id="使用_charCodeAt_修复字符串中出现的已知的非BMP字符">使用 <code>charCodeAt()</code> 修复字符串中出现的已知的非 BMP 字符</h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/charcodeat/index.html:99]:
```suggestion
<h3 id="使用_charcodeat_修复字符串中出现的已知的非_bmp_字符">使用 <code>charCodeAt()</code> 修复字符串中出现的已知的非 BMP 字符</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/codepointat/index.html [hunk 14]
```diff
@@ -24,9 +24,9 @@ <h3 id="返回值">返回值</h3>
 
 <h2 id="描述">描述</h2>
 
-<p>如果在指定的位置没有元素则返回 {{jsxref("undefined")}} 。如果在索引处开始没有UTF-16 代理对，将直接返回在那个索引处的编码单元。</p>
+<p>如果在指定的位置没有元素则返回 {{jsxref("undefined")}} 。如果在索引处开始没有 UTF-16 代理对，将直接返回在那个索引处的编码单元。</p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/codepointat/index.html:27]:
```suggestion
<p>如果在指定的位置没有元素则返回 {{jsxref("undefined")}}。如果在索引处开始没有 UTF-16 代理对，将直接返回在那个索引处的编码单元。</p>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/codepointat/index.html [hunk 15]
```diff
@@ -38,7 +38,7 @@ <h3 id="使用_codePointAt()">使用 <code>codePointAt()</code></h3>
 'XYZ'.codePointAt(42); // undefined
 </pre>
 
-<h2 id="替补支持(Polyfill)">替补支持(Polyfill)</h2>
+<h2 id="替补支持(Polyfill)">替补支持 (Polyfill)</h2>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/codepointat/index.html:41]:
```suggestion
<h2 id="替补支持（polyfill）">替补支持（Polyfill）</h2>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/fixed/index.html [hunk 16]
```diff
@@ -9,7 +9,7 @@
 ---
 <div>{{JSRef}} {{deprecated_header}}</div>
 
-<p>fixed()方法创建了一个&lt;tt&gt;标签元素将字符串包裹起来，从而让这个字符串里面的内容具有固定间距。</p>
+<p>fixed() 方法创建了一个&lt;tt&gt;标签元素将字符串包裹起来，从而让这个字符串里面的内容具有固定间距。</p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/fixed/index.html:12]:
```suggestion
<p>fixed() 方法创建了一个 &lt;tt&gt; 标签元素将字符串包裹起来，从而让这个字符串里面的内容具有固定间距。</p>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/fixed/index.html [hunk 17]
```diff
@@ -22,13 +22,13 @@ <h3 id="Return_value">Return value</h3>
 
 <h2 id="描述">描述</h2>
 
-<p><code>fixed()</code> 方法将一个字符串包裹在&lt;tt&gt;&lt;/tt&gt;标签中，比如: <code>"&lt;tt&gt;str&lt;/tt&gt;"</code>.</p>
+<p><code>fixed()</code> 方法将一个字符串包裹在&lt;tt&gt;&lt;/tt&gt;标签中，比如：<code>"&lt;tt&gt;str&lt;/tt&gt;"</code>.</p>
 
 <h2 id="举例">举例</h2>
 
-<h3 id="使用fixed()">使用fixed()</h3>
+<h3 id="使用fixed()">使用 fixed()</h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/fixed/index.html:29]:
```suggestion
<h3 id="使用_fixed">使用 fixed()</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/fontsize/index.html [hunk 18]
```diff
@@ -32,7 +32,7 @@ <h2 id="描述">描述</h2>
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用fontsize()方法"><code>使用fontsize()方法</code></h3>
+<h3 id="使用_fontsize_方法"><code>使用 fontsize() 方法</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/fontsize/index.html:35]:
```suggestion
<h3 id="使用_fontsize_方法">使用 <code>fontsize()</code> 方法</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/match/index.html [hunk 19]
```diff
@@ -87,27 +87,27 @@ <h3 id="Example_Using_global_and_ignore_case_flags_with_match">例子：<code>ma
 console.log(matches_array);
 // ['A', 'B', 'C', 'D', 'E', 'a', 'b', 'c', 'd', 'e']</pre>
 
-<h3 id="使用match，不传参数"><code>使用match()，不传参数 </code></h3>
+<h3 id="使用match，不传参数"><code>使用 match()，不传参数 </code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/match/index.html:90]:
```suggestion
<h3 id="使用_match，不传参数">使用 <code>match()</code>，不传参数</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/repeat/index.html [hunk 20]
```diff
@@ -40,7 +40,7 @@ <h3 id="Exceptions">Exceptions</h3>
  <li>{{jsxref("Errors/Resulting_string_too_large", "RangeError")}}: 重复次数必须小于 infinity，且长度不会大于最长的字符串。</li>
 </ul>
 
-<h2 id="兼容补丁Polyfill">兼容补丁(Polyfill)</h2>
+<h2 id="兼容补丁Polyfill">兼容补丁 (Polyfill)</h2>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/repeat/index.html:43]:
```suggestion
<h2 id="兼容补丁（polyfill）">兼容补丁（Polyfill）</h2>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/small/index.html [hunk 21]
```diff
@@ -21,7 +21,7 @@ <h2 id="描述">描述</h2>
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用small()函数"><code>使用small()函数</code></h3>
+<h3 id="使用small()函数"><code>使用 small() 函数</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/small/index.html:24]:
```suggestion
<h3 id="使用_small_函数">使用 <code>small()</code> 函数</h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/tolocalelowercase/index.html [hunk 22]
```diff
@@ -17,7 +17,7 @@ <h3 id="参数">参数</h3>
 
 <dl>
  <dt><code>locale</code> {{optional_inline}}</dt>
- <dd>参数 <code>locale</code> 指明要转换成小写格式的特定语言区域。 如果以一个数组 {{jsxref("Array")}}形式给出多个locales,  最合适的地区将被选出来应用（参见<a href="https://tc39.github.io/ecma402/#sec-bestavailablelocale">best available locale</a>）。默认的locale是主机环境的当前区域(locale)设置。</dd>
+ <dd>参数 <code>locale</code> 指明要转换成小写格式的特定语言区域。 如果以一个数组 {{jsxref("Array")}}形式给出多个 locales,  最合适的地区将被选出来应用（参见<a href="https://tc39.github.io/ecma402/#sec-bestavailablelocale">best available locale</a>）。默认的 locale 是主机环境的当前区域 (locale) 设置。</dd>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/tolocalelowercase/index.html:20]:
```suggestion
 <dd>参数 <code>locale</code> 指明要转换成小写格式的特定语言区域。 如果以一个数组 {{jsxref("Array")}} 形式给出多个 locales,  最合适的地区将被选出来应用（参见<a href="https://tc39.github.io/ecma402/#sec-bestavailablelocale">best available locale</a>）。默认的 locale 是主机环境的当前区域 (locale) 设置。</dd>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/tolocalelowercase/index.html [hunk 23]
```diff
@@ -33,11 +33,11 @@ <h3 id="Exceptions">Exceptions</h3>
 
 <h2 id="描述">描述</h2>
 
-<p><code>toLocaleLowerCase()</code> 方法返回根据任意区域语言大小写映射集而转换成小写格式的字符串。<code>toLocaleLowerCase()</code> 并不会影响字符串原本的值。在大多数情况下，该方法和调用 {{jsxref("String.prototype.toLowerCase()", "toLowerCase()")}}的结果相同，但是在某些区域环境中，比如土耳其语，它的大小写映射并不遵循在Unicode中的默认的大小写映射，因此会有一个不同的结果。</p>
+<p><code>toLocaleLowerCase()</code> 方法返回根据任意区域语言大小写映射集而转换成小写格式的字符串。<code>toLocaleLowerCase()</code> 并不会影响字符串原本的值。在大多数情况下，该方法和调用 {{jsxref("String.prototype.toLowerCase()", "toLowerCase()")}}的结果相同，但是在某些区域环境中，比如土耳其语，它的大小写映射并不遵循在 Unicode 中的默认的大小写映射，因此会有一个不同的结果。</p>
 
 <h2 id="例子">例子</h2>
 
-<h3 id="使用toLocaleLowerCase"><code>使用toLocaleLowerCase()</code></h3>
+<h3 id="使用toLocaleLowerCase"><code>使用 toLocaleLowerCase()</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/tolocalelowercase/index.html:40]:
```suggestion
<h3 id="使用_tolocalelowercase">使用 <code>toLocaleLowerCase()</code></h3>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/tosource/index.html [hunk 24]
```diff
@@ -17,20 +17,20 @@ <h2 id="描述">描述</h2>
 
 <p><code>toSource()</code> 方法返回以下值：</p>
 
-<p>对于内建对象 {{jsxref("String")}} , <code>toSource()返回以下值说明源码不可用：</code></p>
+<p>对于内建对象 {{jsxref("String")}} , <code>toSource() 返回以下值说明源码不可用：</code></p>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/tosource/index.html:20]:
```suggestion
<p>对于内建对象 {{jsxref("String")}}，<code>toSource()</code> 返回以下值说明源码不可用：</p>
```

File: files/zh-cn/web/javascript/reference/global_objects/syntaxerror/index.html [hunk 25]
```diff
@@ -24,23 +24,23 @@ <h3 id="Parameters">参数</h3>
...
- <dd>可选的. 包含引发异常的代码的行号</dd>
+ <dd>可选的。包含引发异常的代码的行号</dd>
 </dl>
 
 <h2 id="Properties">属性</h2>
 
 <dl>
  <dt>{{jsxref("SyntaxError.prototype")}}</dt>
- <dd><code>允许SyntaxError</code>对象添加属性.</dd>
+ <dd><code>允许 SyntaxError</code>对象添加属性。</dd>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/syntaxerror/index.html:38]:
```suggestion
 <dd>允许 <code>SyntaxError</code>对象添加属性。</dd>
```

File: files/zh-cn/web/javascript/reference/global_objects/string/@@iterator/index.html [hunk 26]
```diff
@@ -17,11 +17,11 @@ <h2 id="语法">语法</h2>
 
 <h3 id="返回值">返回值</h3>
 
-<p>一个新的Iterator对象。</p>
+<p>一个新的 Iterator 对象。</p>
 
 <h2 id="示例">示例</h2>
 
-<h3 id="使用iterator"><code>使用[@@iterator]()</code></h3>
+<h3 id="使用iterator"><code>使用 [@@iterator]()</code></h3>
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/string/@@iterator/index.html:24]:
```suggestion
<h3 id="使用_iterator">使用 <code>[@@iterator]()</code></h3>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Pull request merged**